operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md

```diff
@@ -202,50 +202,17 @@ Verify that the Lustre file system is available from the management cluster.
     - ncn-w001
     - ncn-w002
     - ncn-w003
+    - ncn-w004
 
     Are the above NCN groupings correct? [yes,no] yes
+    Executing step: Ensure containerd is running and enabled on all Kubernetes NCNs.
+    Executing step: Ensure etcd is running and enabled on all Kubernetes manager NCNs.
+    Executing step: Start and enable kubelet on all Kubernetes NCNs.
+    Waiting up to 300 seconds for the Kubernetes API to become available
+    The Kubernetes API is currently unreachable.
+    Kubernetes API is available
     ```
 
-    The `sat bootsys boot` command may fail with a message like the following:
```

Comment:
We still need something about the possible Ceph failures, but it needs to move to the `sat bootsys boot --stage ncn-power` command and needs some slight modification to account for the fact that deep scrub is now allowed to pass the health check and that there may be a prompt allowing the admin to continue despite a failed health check.

Will you please add this back in for now, so that we remember to include it in the ncn-power stage description?

Reply:
Sure